.github/workflows/full-checks.yml

```diff
@@ -0,0 +1,202 @@
+name: Full Checks
+
+on:
+  schedule:
+    - cron: '0 1 * * *' # run at 1AM UTC
+  workflow_dispatch:
+
+env:
+  HOMEBREW_NO_INSTALL_CLEANUP=1: 1 # Disable cleanup for homebrew, we don't need it on CI
+
+jobs:
+  build-and-test-release:
+    name: Test LLC + UI (Release)
+    runs-on: macos-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set build image var
+      run: echo "ImageVersion=$ImageVersion"  >> $GITHUB_ENV
+    - name: Cache RubyGems
+      uses: actions/cache@v2
+      id: rubygem-cache
+      with:
+        path: vendor/bundle
+        key: ${{ runner.os }}-${{ env.ImageVersion }}-gem-${{ hashFiles('**/Gemfile.lock') }}
+        restore-keys: ${{ runner.os }}-${{ env.ImageVersion }}-gem-
+    - name: Cache Mint
+      uses: actions/cache@v2
+      id: mint-cache
+      with:
+        path: /usr/local/lib/mint
+        key: ${{ runner.os }}-mint-${{ hashFiles('./Mintfile') }}
+        restore-keys: ${{ runner.os }}-mint-
+    - uses: ./.github/actions/bootstrap
+    - name: Run LLC Tests (Release)
+      run: bundle exec fastlane test_release
+    - name: Run UI Tests (Release)
+      run: bundle exec fastlane test_ui_release device:"iPhone 12"
+    - uses: 8398a7/action-slack@v3
+      with:
+        status: ${{ job.status }}
+        text: "You shall not pass!"
+        fields: message,commit,author,action,eventName,workflow,job,took
+      env:
+        SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+        MATRIX_CONTEXT: ${{ toJson(matrix) }}
```

Comment:
Why don't we add an action for this too?

Reply:
Same as above